Lib/fontbakery/utils.py

```diff
@@ -31,6 +33,27 @@
     PANOSE_Family_Type,
 )
 
+if TYPE_CHECKING:
+    from pathlib import Path
+
+
+def get_resource_file_path(sub_file_path: str) -> Path:
+    """Return the full file path of a resource file inside the fontbakery
+    directory.
+
+    Args:
+        sub_file_path (str): The file path relative to the `fontbakery`
+            directory.
+
+    Returns:
+        Path: The full file path
+    """
+    from importlib.resources import as_file, files
+
+    with as_file(files("fontbakery").joinpath(sub_file_path)) as path:
+        file_path = path
+    return file_path
```

Comment:
technically this is not correct, it will only work when fontbakery is loaded from a filesystem directory (e.g. when installed with pip). It's possible to import python modules from zip files as well, or tools can be used to embded them in a self-contained exe. To support use cases like this you should do what @m4rc1e did in

https://github.com/googlefonts/gftools/pull/1131

Reply:
Thank you! I've updated the PR in #5045. I now return the file contents from the helper function, so no temporary files need to stick around.